### PR TITLE
2020 survey launch

### PR DESCRIPTION
**Pull request by nikomatsakis**
This is meant to be posted on **Thursday, September 10**. The text is adapted from last year's post.

### CODE REVIEW

**Comment by Mark-Simulacrum** [comment]
I feel like maybe we should call out that we're concurrently running the blog requests? Though the form in that post has already received more spam than legitimate blog posts, so I'm a bit hesitant to direct more traffic to it.

**Comment by nikomatsakis** [comment]
Still, I agree that linking to the previous blog post makes sense.

**Comment by nikomatsakis** [comment]
@Mark-Simulacrum I added a sentence

**Comment by jyn514** [comment]
> I feel like maybe we should call out that we're concurrently running the blog requests? Though the form in that post has already received more spam than legitimate blog posts, so I'm a bit hesitant to direct more traffic to it.

Maybe the form should also ask for a github username? The intersection of (people writing a blog post about rust in 2021) and (people without a github) seems pretty small.

File: posts/2020-09-10-survey-launch.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+[vietnamese]: 
+[french]: 
+[italian]: 
+[korean]: 
+[spanish]: 
+[russian]: 
+[traditional chinese]: 
+[swedish]: 
+[german]: 
+[japanese]:
```
Comment (jyn514) [inline posts/2020-09-10-survey-launch.md:52]: These reference links won't match even once you add a URL, they need to start with an uppercase letter.

**Comment by Mark-Simulacrum** [comment]
> Maybe the form should also ask for a github username? The intersection of (people writing a blog post about rust in 2021) and (people without a github) seems pretty small.

I don't know how a GitHub username would help us -- if we really want to talk to the author of the post, we can likely reach out in a number of ways, though I expect we wouldn't have the bandwidth to do so anyway.

**Comment by jyn514** [comment]
Oh, I was mostly referring to cutting down on spam.

File: posts/2020-09-10-survey-launch.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Please help us spread the word by sharing the survey link on your social network feeds, at meetups, around your office, and in other communities.
+
+If you have any questions, please see our [frequently asked questions] or email the Rust Community team at [community-team@rust-lang.org].
+
+Finally, we wanted to thank everyone who helped develop, polish, and test the survey. In particular, we'd like to thank all of the volunteers who worked to provide all of the translations available this year and who will help to translate the results.
+
+[blog.rust-lang.org]: https://blog.rust-lang.org
+[frequently asked questions]: https://forge.rust-lang.org/community/survey-faq.html
+[community-team@rust-lang.org]: mailto:community-team@rust-lang.org
+[2019 survey]: https://blog.rust-lang.org/2020/04/17/Rust-survey-2019.htmlhttps://blog.rust-lang.org/2020/04/17/Rust-survey-2019.html
```
Comment (apiraino) [inline posts/2020-09-10-survey-launch.md:37]:
```suggestion
[2019 survey]: https://blog.rust-lang.org/2020/04/17/Rust-survey-2019.html
```

File: posts/2020-09-10-survey-launch.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "Launching the 2020 State of Rust Survey"
+author: The Rust Community Team
+description: "Hearing from you about the fifth year of Rust"
+---
+
+It's that time again! Time for us to take a look at how the Rust project is doing, and what we should plan for the future. The Rust Community Team is pleased to announce our [2020 State of Rust Survey][survey]! Whether or not you use Rust today, we want to know your opinions. Your responses will help the project understand its strengths and weaknesses and establish development priorities for the future. (If you'd like to give longer form feedback on the Rust roadmap, [we're also collecting blog posts!])
```
Comment (apiraino) [inline posts/2020-09-10-survey-launch.md:8]: > [we're also collecting blog posts!]

I think the intent was to have a link either here: 
https://blog.rust-lang.org/2020/09/03/Planning-2021-Roadmap.html

or here
https://forms.gle/Hv41uA5qJEY89XRm7

File: posts/2020-09-10-survey-launch.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+author: The Rust Community Team
+description: "Hearing from you about the fifth year of Rust"
+---
+
+It's that time again! Time for us to take a look at how the Rust project is doing, and what we should plan for the future. The Rust Community Team is pleased to announce our [2020 State of Rust Survey][survey]! Whether or not you use Rust today, we want to know your opinions. Your responses will help the project understand its strengths and weaknesses and establish development priorities for the future. (If you'd like to give longer form feedback on the Rust roadmap, [we're also collecting blog posts!](https://blog.rust-lang.org/2020/09/03/Planning-2021-Roadmap.html))
+
+Completing this survey should take about 10–15 minutes and is anonymous unless you choose to give us your contact information. We will be accepting submissions for the next two weeks (until September 24th), and we will write up our findings afterwards to [blog.rust-lang.org]. You can also check out [last year’s results][2019 survey].
+
+- [English][survey]
+- [Simplified Chinese]
```
Comment (upsuper) [inline posts/2020-09-10-survey-launch.md:13]: Address of Simplified Chinese is https://wj.qq.com/s2/7111747/269f

Comment (nikomatsakis) [reply posts/2020-09-10-survey-launch.md:13]: added, thanks!

**Pull request merged**